.vscode/settings.json

```diff
@@ -0,0 +1,65 @@
+{
+	// .NET & C# Settings
+	"dotnet.defaultSolution": "Obsidian.sln",
+	"omnisharp.enableEditorConfigSupport": true,
+	"omnisharp.enableRoslynAnalyzers": true,
+	"omnisharp.useModernNet": true,
+	// Editor Settings
+	"editor.formatOnSave": true,
+	"editor.codeActionsOnSave": {
+		"source.organizeImports": "explicit",
+		"source.fixAll": "explicit"
+	},
+	"editor.rulers": [
+		120
+	],
+	"editor.tabSize": 4,
+	"editor.insertSpaces": false,
+	// File Management
+	"files.exclude": {
+		"**/bin": true,
+		"**/obj": true,
+		"**/.vs": true,
+		"**/Generated": true
+	},
```

Comment:
Not necessarily needed, you should have a solution explorer in VSCode too (under file list)

Reply:
I meant to add that and forgot. Adding